common/dna_data/wc_human_dna.txt

```diff
@@ -2699,4 +2699,608 @@ aiden_perenolde_dna = { # by ercarp
+ 		gene_bs_nose_size={ "nose_size_neg" 57 "nose_size_pos" 75 }
+ 		gene_bs_nose_tip_angle={ "nose_tip_angle_neg" 29 "nose_tip_angle_pos" 45 }
+ 		gene_bs_nose_tip_forward={ "nose_tip_forward_neg" 49 "nose_tip_forward_neg" 33 }
+ 		gene_bs_nose_tip_width={ "nose_tip_width_neg" 97 "nose_tip_width_pos" 22 }
+ 		face_detail_cheek_def={ "cheek_def_02" 148 "cheek_def_02" 4 }
+ 		face_detail_cheek_fat={ "cheek_fat_04_pos" 34 "cheek_fat_01_pos" 8 }
+ 		face_detail_chin_cleft={ "chin_dimple" 16 "chin_dimple" 16 }
+ 		face_detail_chin_def={ "chin_def" 36 "chin_def" 36 }
+ 		face_detail_eye_lower_lid_def={ "eye_lower_lid_def" 119 "eye_lower_lid_def" 49 }
+ 		face_detail_eye_socket={ "eye_socket_03" 255 "eye_socket_01" 254 }
+ 		face_detail_nasolabial={ "nasolabial_03" 242 "nasolabial_03" 5 }
+ 		face_detail_nose_ridge_def={ "nose_ridge_def_pos" 135 "nose_ridge_def_pos" 68 }
+ 		face_detail_nose_tip_def={ "nose_tip_def" 150 "nose_tip_def" 36 }
+ 		face_detail_temple_def={ "temple_def" 232 "temple_def" 39 }
+ 		expression_brow_wrinkles={ "brow_wrinkles_01" 142 "brow_wrinkles_03" 41 }
+ 		expression_eye_wrinkles={ "eye_wrinkles_02" 117 "eye_wrinkles_02" 165 }
+ 		expression_forehead_wrinkles={ "forehead_wrinkles_02" 159 "forehead_wrinkles_02" 249 }
+ 		expression_other={ "cheek_wrinkles_left_01" 127 "cheek_wrinkles_left_01" 127 }
+ 		complexion={ "complexion_1" 139 "complexion_2" 237 }
+ 		gene_height={ "normal_height" 122 "normal_height" 139 }
+ 		gene_bs_body_type={ "body_fat_head_fat_low" 106 "body_fat_head_fat_medium" 132 }
+ 		gene_bs_body_shape={ "body_shape_average_clothed" 255 "body_shape_hourglass_full" 0 }
+ 		gene_bs_bust={ "bust_clothes" 130 "bust_shape_3_full" 130 }
+ 		gene_age={ "old_beauty_1" 60 "old_1" 60 }
+ 		gene_eyebrows_shape={ "avg_spacing_lower_thickness" 203 "far_spacing_low_thickness" 184 }
+ 		gene_eyebrows_fullness={ "layer_2_lower_thickness" 143 "layer_2_low_thickness" 174 }
+ 		gene_body_hair={ "body_hair_sparse" 0 "body_hair_sparse" 181 }
+ 		gene_hair_type={ "hair_straight" 129 "hair_straight" 129 }
+ 		gene_baldness={ "no_baldness" 127 "no_baldness" 127 }
+ 		eye_accessory={ "normal_eyes" 166 "normal_eyes" 166 }
+ 		teeth_accessory={ "normal_teeth" 0 "normal_teeth" 0 }
+ 		eyelashes_accessory={ "normal_eyelashes" 107 "normal_eyelashes" 107 }
+ 		gene_bs_ear_lenght={ "floppy_ear_length" 0 "floppy_ear_length" 0 }
+ 		special_eyebrows={ "no_eyebrows" 101 "no_eyebrows" 101 }
+ 		special_eyes={ "no_special_eyes" 0 "no_special_eyes" 0 }
+ 		gene_bs_mouth_tusks={ "mouth_fangs" 0 "mouth_fangs" 0 }
+ 		static_tusks={ "no_static_tusks" 0 "no_static_tusks" 0 }
+ 		gene_bs_ear_tip={ "ear_tip_pos" 0 "ear_tip_pos" 0 }
+ 		gene_bs_fingers={ "fingers_troll" 0 "fingers_troll" 127 }
+ 		gene_skin_value={ "skin_value_pos" 0 "skin_value_neg" 0 }
+ 		gene_race={ "creature_human" 141 "creature_human" 141 }vCX XC
+		}
+    }
+	enabled=yes
+}
+
+aedalas_blackmoore_dna = {
```

Comment:
What you used for a reference? He looks too chubby.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/123140003-de3a1300-d467-11eb-8d24-ee143ee1739b.png)

</details>

Reply:
tru

Reply:
DNA will be reworked on the `dna-2` branch.

62dd73f68a2e1f45710f41f5537d3395321dd192